### DISCUSSION TITLE
Domain redirect loop (www -> @, 307 + 308)

### DISCUSSION THREAD

**Discussion by shiftwork-site** [vercel/community#3821] [Help]
### Summary

I changed my DNS to the following:
A Record | @ | 76.76.21.21 
CNAME Record | qr | cname.vercel-dns.com. 
CNAME Record | www | cname.vercel-dns.com.
CNAME Record | old | ns1.darwin.sui-inter.net.
CNAME Record | old | ns2.darwin.sui-inter.net.

When I loaded shiftwork.cc, either of these two happened:
A. I had a redirect loop going from `www.shiftwork.cc` to `shiftwork.cc` and back.
B. I was displayed with the old website (dns record CNAME ns1.sui-inter.net, reading _SHIFT, UPCOMING LAUNCH ..._)

My desired settings are:
`shiftwork.cc` redirects to `www.shiftwork.cc` (308 permanent)
`www.shiftwork.cc` doesn't redirect (shows website)
Both domains show _Valid configuration_

Now – it's _possible_ that I used to have a 308 permanent redirect from `www.shiftwork.cc` to `shiftwork.cc` that I removed... If that was the cause of the issue, could I fix that?

Any help is much appreciated!

### Example

I've changed the DNS back to the old website for now.
[http://shiftwork.cc](shiftwork.cc)

### Steps to Reproduce

For now, I don't have steps to reproduce as I changed the DNS back.

**Comment by amyegan** [comment]
Hey @shiftwork-site. It looks like your domain is using Vercel nameservers now (https://www.whatsmydns.net/#NS/shiftwork.cc). Are you still having domain record problems, or is it working for you now?
